packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -72,16 +72,23 @@ export interface SliderCommon extends Omit<React.HTMLAttributes<HTMLDivElement>,
 
   /**
    * The max value of the **Slider**.
-   * @default 10
+   * @default 100
    */
   max?: number;
 
   /**
-   * The number of steps that the **Slider's** `value` will increment upon change.
-   * @default 1
+   * The number of steps that the Slider's `value` will increment upon change. When provided, the Slider
+   * will snap to the closest available value.
    */
   step?: number;
 
+  /**
+   * The number of increments that the Slider's value will change by during a key press.
```

Comment:
You should call out here that opting in for this behavior might cause the `Slider` to go outside of snapping values when moved through the keyboard.